cds_ils/importer/XMLRecordToJson.py

```diff
@@ -54,14 +54,18 @@ def dump(self):
             is_deletable = False
 
         init_fields = {}
-        if "im" in marc_record.get("leader", []):
+        if "am" in marc_record.get("leader", []):
+            init_fields.update({"_eitem": {"_type": "e-book"}})
+        elif "im" in marc_record.get("leader", []) or "jm" in marc_record.get(
+            "leader", []
+        ):
```

Comment:
small suggestion just for readability
```suggestion
        leader_tag = marc_record.get("leader", [])
        if "am" in leader_tag:
            init_fields.update({"_eitem": {"_type": "e-book"}})
        elif "im" in leader_tag or "jm" in leader_tag:
```
(also in line 68)